src/System.Data.SqlClient/dir.props

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\dir.props" />
   <PropertyGroup>
-    <AssemblyVersion>4.4.1.0</AssemblyVersion>
+    <AssemblyVersion>4.4.2.0</AssemblyVersion>
```

Comment:
This should become "4.5.0.0" given you are adding new public API.

Reply:
Thanks. Updated it to 4.5.0.0